main.py

```
@@ -1089,9 +1089,10 @@ async def _handle_dose_taken(self, query, context):
         reminder_key = f"{user.id}_{medicine_id}"
         medicine_scheduler.reminder_attempts[reminder_key] = 0
 
-        await query.edit_message_text(
-            f"{config.EMOJIS['success']} נטילת התרופה אושרה!\n" f"מלאי נותר: {new_count if medicine else 'לא ידוע'} כדורים"
-        )
+        confirmation = f"{config.EMOJIS['success']} נטילת התרופה אושרה!"
+        if medicine and await tracks_inventory(query.from_user.id):
+            confirmation += f"\nמלאי נותר: {new_count} כדורים"
```

Ruff (0.16.1)

[warning] String contains ambiguous `י` (HEBREW LETTER YOD). Did you mean ``` (GRAVE ACCENT)?

(RUF001)